lib/install/management/appliance.go

```diff
@@ -397,6 +409,12 @@ func (d *Dispatcher) findApplianceByID(conf *config.VirtualContainerHostConfigSp
 		return nil, err
 	}
 	vmm = vm.NewVirtualMachine(d.op, d.session, ovm.Reference())
+
+	element, err := d.session.Finder.Element(d.op, vmm.Reference())
```

Comment:
I am thinking we will want this to become a function from the dispatcher which creates a singleton private var. so if private var == nil get pointer to VMFolder and set private var... etc...